docs/basics/part-1-a-simple-earthfile.md

```diff
@@ -1,10 +1,17 @@
 # A simple Earthfile
 
-Earthfiles are always named `Earthfile`, regardless of their location in the codebase.
+All the magic of Earthly happens in the Earthfile. Earthfiles are always named `Earthfile`, regardless of their location in the codebase. Below you'll find several example Earthfiles. 
+
+- [Go](#go) 
+- [JavaScript](#javascript) 
+- [Java](#java)
+- [Python](#python)
+
+
+### Go
 
 {% method %}
 {% sample lang="Go" %}
```

Comment:
Would you mind removing these since they no longer work?

Reply:
Removed 👍